content/docs/command-reference/metrics/index.md

```diff
@@ -27,21 +21,76 @@ positional arguments:
 In order to track metrics associated to machine learning experiments, DVC has
 the ability to mark a certain stage <abbr>outputs</abbr> as files containing
 metrics to track (see the `--metrics` option of `dvc run`). Metrics are
-project-specific floating-point values e.g. `AUC`, `ROC`, etc.
-
-Supported file formats: JSON. Metrics can be organized in a tree hierarchy in a
-JSON file. DVC addresses the metrics by the tree path.
+project-specific floating-point or integer values e.g. `AUC`, `ROC`,
+`False-Positives` etc.
+
+[Show](/doc/command-reference/metrics/show), and
+[diff](/doc/command-reference/metrics/diff), commands are available to manage
+<abbr>DVC project</abbr> metrics.
+
+## Formats and metrics types
+
+Supported file formats: JSON and YAML. Metrics can be organized in a tree
+hierarchy in a JSON file or a YAML file. DVC addresses the metrics by the tree
+path. In the example below five metrics are presented: `train.accuracy`,
+`train.loss`, `train.TN`, `train.FP` and `time_real`.
```

Comment:
I always wanted to improve this paragrpah:
```suggestion
Metrics can be organized in a tree hierarchy in a JSON or YAML file. DVC
addresses specific metrics by the tree path. In the JSON example below,
five metrics are presented: `train.accuracy`, `train.loss`, `train.TN`, `train.FP`
and `time_real`.
```

Reply:
Also applying this and other matching text updates to current dvc plot cmd ref...